pkgs/development/libraries/libgudev/default.nix

```diff
@@ -25,6 +25,15 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-YSZqsa/J1z28YKiyr3PpnS/f9H2ZVE0IV2Dk+mZ7XdE=";
   };
 
+  patches = lib.optionals stdenv.hostPlatform.isMusl [
+    ./libgudev-disable-test-double.patch # disable a tests that requires a correct implementation of locales
+  ];
```

Comment:
What if we took the [patch](https://gitlab.gnome.org/GNOME/libgudev/-/merge_requests/31) I sent upstream that skips this on musl but runs it otherwise?  Then we could just unconditionally apply it.

Reply:
Oh that would be much better, I wasn't aware of its existence

Reply:
Probably best to commit the patch rather than fetchpatch-ing, because it might not stick around if they rebase merge and I delete the branch.

(Other patch file also needs to be removed now, btw.)

Reply:
Done